react/features/base/tracks/actions.js

```diff
@@ -155,6 +158,11 @@ export function replaceLocalTrack(oldTrack, newTrack, conference) {
                                     ? setVideoMuted
                                     : setAudioMuted;
 
+                            sendEvent(
+                                `replacetrack.${newTrack.getType()}.mute`);
```

Comment:
Instead of ".mute" shouldn't we add the newTrack.isMuted() state?

Reply:
Yes.